pages/index.js

```diff
@@ -307,8 +307,7 @@ const Home = ({ locale, onLocale, isMobile, articles, seo, navLinks }) => {
                       <Image
                         src={article.cardCover?.url || article.cover?.url || assetRootPath('/images/logos/origin-press.svg')}
                         alt='Origin Protocol'
-                        layout='fill'
-                        objectFit='cover'
+                        width='640' height='312'
```

Comment:
How does that look on devices with a small screen, like mobile?

Reply:
fills out to full width, nextjs/Image resizes while keeping image ratio to container width